docs/source/markdown/podman-exec.1.md.in

```diff
@@ -95,6 +107,18 @@ Execute command but do not attach to the exec session leaving the command runnin
 $ podman exec -d ctrID find /path/to/search -name yourfile
 ```
 
+Execute command with resource limits (requires crun >= 1.9 or runc >= 1.2):
```

Comment:
the OCI runtimes do not have any support for exec and cgroups limits, so this is wrong